node/runtime/pangolin/src/pallets/bridge_parachains.rs

```diff
@@ -4,13 +4,28 @@ pub use pallet_bridge_parachains::{
 
 // --- darwinia-network ---
 use crate::*;
+use bp_polkadot_core::parachains::{ParaId, PARAS_PALLET_NAME};
 use pallet_bridge_parachains::Config;
 
+frame_support::parameter_types! {
+	pub const ParasPalletName: &'static str = PARAS_PALLET_NAME;
+	// TODO: Is it okay to use Everything here
+	pub GetTenFirstParachains: Vec<ParaId> = (0..10).map(ParaId).collect();
+}
+
 impl Config<WithRococoParachainInstance> for Runtime {
 	type BridgesGrandpaPalletInstance = WithRococoGrandpa;
+	type Event = Event;
 	type HeadsToKeep = RococoHeadersToKeep;
+	type ParasPalletName = ParasPalletName;
+	type TrackedParachains = frame_support::traits::Everything;
```

Comment:
@jiguantong Please help check for this, Is `Everything` okay in our parachain scenarios?

Reply:
@fewensa It's used [here](https://github.com/darwinia-network/darwinia-messages-substrate/blob/main/modules/parachains/src/lib.rs#L273). And related upstream pr is https://github.com/paritytech/parity-bridges-common/pull/1432.
Should we track `Everything` here or a specific parachain? like 2105(pangolin parachain)

Reply:
Nope, it's not used by the bridger.

Reply:
@AsceticBear Leave it as `Everything` for now